index.html

```diff
@@ -1391,19 +1415,28 @@ <h3>Selective Disclosure</h3>
 
         <p>
 Selective disclosure is the ability of a <a>holder</a> to select which elements
-from a <a>verifiable credential</a> to share with a <a>verifier</a>. There are
-a couple of different methods which support selective disclosure:
+from a <a>verifiable credential</a> to share with a <a>verifier</a>, without
```

Comment:
`which elements from a`
-> `some elements of a`

Reply:
addressed in [a2642eb](https://github.com/w3c/vc-imp-guide/pull/14/commits/a2642eb58336bcc4303270b72f9e3ffcc8cd74bc)